include/eve/module/arithmetic/pedantic/ieee.hpp

```diff
@@ -0,0 +1,16 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
```

Comment:
This file should not exist. 
If IEEE is in arithmetic, those thing should be in arithmetic/pedantic.hpp.

Reply:
suppressed